binary_transparency/firmware/cmd/usb_armory/image_builder/build.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
A short description (one or two liner) as to what the script does in header, will be helpful!
Also, should we not add the license header to this file also??

Reply:
done